client/src/shared/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Link from 'next/link';
+
+const SideBar: React.FunctionComponent = () => {
+  return (
+    <div className="space-y-9">
+      <div>
+        <div>
+          <h1 className="text-lg pb-2 font-medium">
+            Quick Links |{' '}
```

Comment:
- what the empty single quote for?

Reply:
I think it is the VScode's spacing sir, I encounter this one before I you mix text and <Link>